source/fundamentals/builders/aggregates.txt

```diff
@@ -110,6 +107,34 @@ into a new field called ``rating``, effectively renaming the field.
    :dedent:
 
 
+Documents
+---------
+
+Use the ``documents()`` method to create a
+:manual:`$documents </reference/operator/aggregation/documents/>`
+pipeline stage that returns literal documents from input values.
+
+.. important::
+
+   If you use a ``$documents`` stage in an aggregation pipeline, it must be the first 
+   stage in the pipeline.
+
+The following example creates a pipeline stage that creates 
+sample documents with a ``title`` field: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/builders/AggBuilders.java
+   :start-after: // begin documents
+   :end-before: // end documents
+   :language: java
+   :dedent:
+
+.. tip::
+
+   If you use the ``documents()`` method to provide the input to an aggregation pipeline, 
+   you can call the ``aggregate()`` method on a database instead of on a 
+   collection.
```

Comment:
(Ah, I see that you do mention what I stated in an earlier comment.)

Consider whether this is a tip, since this is not just a "you may/can", but a "you must", like the "must be first" requirement (iirc, an error is produced if $documents is used with a collection).

Reply:
I wasn't aware this was a requirement rather than an option. Updated to an Important box.